docs/docs/user/what-is-kpops.md

```diff
@@ -1,7 +1,26 @@
 # What is KPOps?
 
-KPOps is a CLI tool that allows deploying and destroying consecutive Kafka Streams applications, Producers, and Kafka 
-Connectors using an easy-to-read and write pipeline definition. KPOps comes with various abstractions that can simplify
-configuring multiple pipelines and steps within pipelines by sharing configuration between different applications,
-like producers or streaming applications. In addition, Kafka apps and its resources can be reset or cleaned up 
-again with a single command.
+With a couple of easy commands in the shell and a [`pipeline.yaml`](#example) of under 30 lines, [KPOps](/) can not only [`deploy`](/user/references/cli-commands/#deploy) a Kafka pipeline[^1] to a Kubernetes cluster, but also [`reset`](/user/references/cli-commands/#reset), [`clean`](/user/references/cli-commands/#clean) or [`destroy`](/user/references/cli-commands/#destroy) it!
+[^1]:
+     A Kafka pipeline can consist of consecutive [streaming applications](/user/references/components/#streamsapp), [producers](/user/references/components/#producer), and [connectors](/user/references/defaults/#kafkaconnector).
+
+## Key features
+
+- Easy-to-read and -write pipeline definitions
+- Deploy pipelines with minimal effort and without sacrificing configurability
+- Reset or destroy pipelines gracefully
+- Reset or clean apps again with a single command
+- Set [defaults](/user/references/defaults) to share settings within and across pipelines
```

Comment:
@disrupted WDYT?
```suggestion
- **Deploy Kafka apps to Kubernetes**: KPOps allows to deploy consecutive Kafka Streams applications and producers using an easy-to-read and write pipeline definition.
- **Manage Kafka Connectors**: KPOps connects with your Kafka connect cluster and applies, validates, and deletes your connector configs.
- **Configure multiple pipelines and steps**: KPOps has various abstractions that simplify configuring multiple pipelines and steps within pipelines by sharing configuration between different applications, like producers or streaming applications.
- **Handle your topics and schemas**: KPOps not only creates and deletes your topics but also registers and deletes your schemas.
- **Manage the lifecycle of your components**: KPOps not only terminates your pipeline components (i.e., Kafka Streams applications) from your Kubernetes cluster, but it cleans up the (Kafka) component-related states (i.e., removing/resetting offset of consumer groups).
```

Reply:
@raminqaf looks great!

Reply:
I don't like the repeating "not only" and I am not sure i **manage the lifecycle of your components** is an accurate title for that point as it only talks about destruction.

```suggestion
- **Deploy Kafka apps to Kubernetes**: KPOps allows to deploy consecutive Kafka Streams applications and producers using an easy-to-read and write pipeline definition.
- **Manage Kafka Connectors**: KPOps connects with your Kafka connect cluster and applies, validates, and deletes your connector configs.
- **Configure multiple pipelines and steps**: KPOps has various abstractions that simplify configuring multiple pipelines and steps within pipelines by sharing configuration between different applications, like producers or streaming applications.
- **Handle your topics and schemas**: KPOps not only creates and deletes your topics but also registers and deletes your schemas.
- **Remove your components**: KPOps removes your pipeline components (i.e., Kafka Streams applications) from the Kubernetes cluster AND cleans up the (Kafka) component-related states (i.e., removing/resetting offset of consumer groups).
- **Preview your pipeline changes**: With the KPOps dry-run, you can ensure your pipeline definition is set up correctly. This helps to minimize downtime and prevent potential errors or issues that could impact your production environment.
```

Reply:
I like it now even more. How about we change **Remove your components** to **Clean termination of Kafka components**

Reply:
Even better! :) 

@disrupted wdyt?

Reply:
great improvements all around, go for it!